env/src/test/java/com/sap/cloud/security/config/cf/CFEnvParserTest.java

```diff
@@ -34,33 +35,33 @@ public CFEnvParserTest() throws IOException {
 
 	@Test
 	public void getClientId() {
-		assertThat(cut.getClientId()).isEqualTo("T000310");
+		Assertions.assertThat(cut.getClientId()).isEqualTo("T000310");
```

Comment:
this seems to be a merge error and I do not expect any change on existing classes here in this PR beside of Environments.